README.md

```diff
@@ -22,11 +22,15 @@ Daily updated from their sources, whether it is ISO or IANA directly, or a maint
 
 To start right away, run the following command in your composer project;
 
-```composer require prinsfrank/standards```
+```bash
```

Comment:
I didn't know that the short notation resulted in no copy button, thanks, this makes sense!